python/py-service_identity/Portfile

```diff
@@ -37,6 +37,14 @@ if {${name} ne ${subport}} {
                         port:py${python.version}-attrs
 
     if {${python.version} eq 27} {
+        # last working version for py27
+        version     21.1.0
+        master_sites https://files.pythonhosted.org/packages/09/2e/26ade69944773df4748c19d3053e025b282f48de02aad84906d34a29d28b
+        distname    service-identity-${version}
```

Comment:
Instead of setting `master_sites` and `distname`, do `python.rootname service-identity`

Reply:
Thanks done.